src/helpers/links.js

```diff
@@ -111,7 +111,7 @@ const openLink = function(event, _attrs) {
 		OCA.Viewer.open({ path })
 		return
 	}
-	if (query?.fileId) {
+	if (htmlHref.match(/apps\/files\//) && query?.fileId) {
```

Comment:
Is this check enough? Shouldn't we check it's also a link to the same instance where we are right now?
Maybe in another PR...

Reply:
Yeah, let's do that in a separate PR :blush: